package.json

```diff
@@ -85,12 +84,14 @@
     "dompurify": "3.0.1",
     "epubjs": "0.3.93",
     "escape-html": "1.0.3",
+    "event-target-polyfill": "github:ThaUnknown/event-target-polyfill",
```

Comment:
I realize this should be temporary, but I'm not a fan of installing directly from github here. It's not a great solution, but if it's not available on npm perhaps we should include it in `src/legacy` with some of the other polyfills for now. :man_shrugging:

Reply:
no followup on this, and since this is already done previously in this package i'm gonna assume its fine